packages/upgrade/static/src/downgrade_module.ts

```diff
@@ -361,7 +361,7 @@ export function downgradeModule<T>(moduleOrBootstrapFn: NgModuleFactory<T>): str
  *   {@link PlatformRef#bootstrapmodulefactory `bootstrapModuleFactory()`} to bootstrap the
  *   downgraded modules, each one is considered a "root" module. As a consequence, a new instance
  *   will be created for every injectable provided in `"root"` (via
- *   {@link Injectable#providedIn `providedIn`}).
+ *   [providedIn](/api/core/Injectable#providedIn)).
```

Comment:
```suggestion
 *   [`providedIn`](/api/core/Injectable#providedIn)).
```

Make sure to include the backticks when referencing a symbol that is not a `@see` link.